GarNet/garnet.py

```diff
@@ -226,23 +153,34 @@ def TF_regression(motifs_and_genes_file_or_dataframe, expression_file, output_di
 		# Occasionally there's only one gene associated with a TF, which we can't fit a line to.
 		if len(expression_profile) < 5: continue
 
+		# This allows heavier points to be visualized on the top instead of being hidden by long distance points 
```

Comment:
Would you mind explaining this just a little more? I'm not sure I get it.

Reply:
Hmm.. it looks like the points are also behind the grid lines. Doesn't matter much, but do you think it would be easy to fix in the scope of this PR?

Reply:
Yes, it can be fixed by uncommenting the following line I think.